examples/rust/files/chunk_sender/send_file.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# Usage: ./send_file.sh [path/to/file]
+# If no file is provided, defaults to data/test.yaml
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+WORKSPACE_ROOT="$SCRIPT_DIR/../../../../"
+
+DAEMON_PATH="$WORKSPACE_ROOT/target/release/aranya-daemon"
+SENDER_PATH="$WORKSPACE_ROOT/examples/rust/files/sender/target/release/sender"
```

Comment:
This is the path for the other example:
```suggestion
SENDER_PATH="$WORKSPACE_ROOT/examples/rust/files/chunk_sender/target/release/sender"
```